test/distributions/test_transforms.py

```diff
@@ -0,0 +1,365 @@
+from numbers import Number
+
+import pytest
+
+import torch
+from torch.autograd.functional import jacobian
+from torch.distributions import Dirichlet, Normal, TransformedDistribution, constraints
+from torch.distributions.utils import tril_matrix_to_vec, vec_to_tril_matrix
+from torch.distributions.transforms import (AbsTransform, AffineTransform, ComposeTransform,
```

Comment:
lint: `transforms -> utils`

Reply:
Interestingly, linter didn't complain. My current IDE setup isn't automatically correcting lint errors, which I will fix shortly. Thanks for the heads up.